### PR TITLE
feat: move MinBidPrice and MatchedPrice from BaseAuction to BatchAuction

### LINKED ISSUE #124: feat: move MinBidPrice/MatchedPrice from BaseAuction to BatchAuction
## Summary

- `MinBidPrice` and `MatchedPrice` do not have any functionality in a fixed price auction. It affects only a batch auction.
- In order to make this clear, it is proposed to move the variable of `MinBidPrice` and `MatchedPrice` from `BaseAuction` to `BatchAuction` 

## Problem Definition

- `MinBidPrice` and `MatchedPrice` do not have any functionality in a fixed price auction. It affects only a batch auction.

## Proposal

- To move `MinBidPrice` and `MatchedPrice` in `BaseAuction` to `BatchAuction`

## Tasks

- 

## References

-
____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

### PR DESCRIPTION
**Pull request by jeonghoyeo7**
## Description

closes: #124 

## Tasks

- [x] Move `MinBidPrice` from `BaseAuction` to `BatchAuction`
- [x] Move `BatchedPrice` from `BaseAuction` to `BatchAuction`
- [x] Modify the related test codes
- [x] Modify the related specs

## References

- 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Appropriate labels applied
- [ ] Targeted PR against correct branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://go.dev/blog/godoc).
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: docs/Tutorials/README.md [hunk 1]
```diff
@@ -14,4 +14,7 @@ This section contains Tutorials documentation for fundraising module.
 
 Tutorials Documentation will come in various forms:
 
-* [Demo](./demo) - contains any type of demo or presentation for fundraising module
\ No newline at end of file
+* [Demo](./demo) - contains any type of demo or presentation for fundraising module
```
Comment (jeonghoyeo7) [inline docs/Tutorials/README.md]: no changes, but only spaces. can be deleted.

File: x/fundraising/keeper/auction.go [hunk 2]
```diff
@@ -259,6 +256,8 @@ func (k Keeper) CreateBatchAuction(ctx sdk.Context, msg *types.MsgCreateBatchAuc
 			sdk.NewAttribute(types.AttributeKeyStartTime, auction.GetStartTime().String()),
 			sdk.NewAttribute(types.AttributeKeyEndTime, msg.EndTime.String()),
 			sdk.NewAttribute(types.AttributeKeyAuctionStatus, auction.GetStatus().String()),
+			sdk.NewAttribute(types.AttributeKeyMinBidPrice, msg.MinBidPrice.String()),
```
Comment (jeonghoyeo7) [inline x/fundraising/keeper/auction.go:259]: @kogisin Please let me know the necessity of this change. I just added in order to fill this with additional variables.

**Review by jaybxyz — CHANGES_REQUESTED** [review]
Can you check the following code lines?

1. https://github.com/tendermint/fundraising/blob/main/x/fundraising/module_test.go#L51
2. https://github.com/tendermint/fundraising/blob/main/x/fundraising/client/cli/utils.go#L18
3. https://github.com/tendermint/fundraising/blob/main/x/fundraising/client/cli/utils_test.go#L19
4. https://github.com/tendermint/fundraising/blob/main/x/fundraising/client/cli/utils_test.go#L63
5. https://github.com/tendermint/fundraising/blob/main/x/fundraising/spec/04_messages.md#msgcreatefixedpriceauction

File: x/fundraising/keeper/auction.go (hunk 2 above)
Comment (jaybxyz) [reply x/fundraising/keeper/auction.go:259]: Yes, I think it is necessary to emit `MinBidPrice` event.

**Comment by jeonghoyeo7** [comment]
@kogisin I checked the lines you commented. Also, I modified `abci_test.go` for minBidPrice.

Besides this, please check 
https://github.com/tendermint/fundraising/blob/8e1a377e5437bb0f78a299e5850058fe2b85fb5d/x/fundraising/client/cli/utils.go#L58
Isn't `EndTime` actually `EndTimes` for plural?

File: x/fundraising/keeper/auction.go [hunk 3]
```diff
@@ -256,8 +256,6 @@ func (k Keeper) CreateBatchAuction(ctx sdk.Context, msg *types.MsgCreateBatchAuc
 			sdk.NewAttribute(types.AttributeKeyStartTime, auction.GetStartTime().String()),
 			sdk.NewAttribute(types.AttributeKeyEndTime, msg.EndTime.String()),
 			sdk.NewAttribute(types.AttributeKeyAuctionStatus, auction.GetStatus().String()),
-			sdk.NewAttribute(types.AttributeKeyMinBidPrice, msg.MinBidPrice.String()),
-			sdk.NewAttribute(types.AttributeKeyMatchedPrice, matchedPrice.String()),
```
Comment (jaybxyz) [inline x/fundraising/keeper/auction.go]: I think there was some misunderstanding. These emit events don't necessarily be removed

**Comment by jaybxyz** [comment]
Great work! LGTM other than the removed emit events above.

**Pull request merged**